### PR TITLE
[plugin.video.tyt] 1.0.0

### PR DESCRIPTION
**Pull request by lottaboost**

### CODE REVIEW

**Comment by learningit** [comment]
You need to add a GPL v2 or v3 license.

File: plugin.video.tyt/main.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+_handle = int(sys.argv[1])
+_url = sys.argv[0]
+xbmcplugin.setContent(_handle, 'tvshows')
+addon_folder = os.path.join(xbmc.translatePath( "special://profile/addon_data/" ), PLUGIN_ID)
+
+settings = xbmcaddon.Addon(id=PLUGIN_ID)
+user_name = settings.getSetting("username")
+user_pwd = settings.getSetting("password")
+COOKIE_FILE = 'special://home/addons/{0}/resources/cookie.txt'.format(PLUGIN_ID)
+cookie = {}
```
Comment (learningit) [inline plugin.video.tyt/main.py]: Please remove this it's not used and is confusing as it looks like you will write into the add-on directory brother than user_data

File: plugin.video.tyt/addon.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+<requires>
+  <import addon="xbmc.python" version="2.24.0"/>
+</requires>
+<extension point="xbmc.python.pluginsource" library="main.py">
+  <provides>video</provides>
+</extension>
+<extension point="xbmc.addon.metadata">
+  <summary lang="en">The Young Turks Kodi Plugin for members.</summary>
+  <description lang="en">Make Sure to set your username and password in addon settings.  All videos that are accessible from the website are playable.  If there is an error, the website hasn't migrated the video from GBox, and won't play on the website either.  Change the view setting in left tab to media info to get video summaries.</description>
+  <disclaimer lang="en">The Young Turks Kodi Plugin for members.</disclaimer>
```
Comment (learningit) [inline plugin.video.tyt/addon.xml]: Please add license and website entries.

**Comment by learningit** [comment]
Thanks. One last thing, you need to squash all the commits down into a single commit. See https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit for how to do that. Should be good to go after that.

**Comment by lottaboost** [comment]
When I go to git "rebase -i HEAD~3" I get "Cannot rebase: You have unstaged changes.
Please commit or stash them."  The response I get if I do "git status" is 
[status.txt](https://github.com/xbmc/repo-plugins/files/427796/status.txt)

**Comment by learningit** [comment]
Firstly, you are not a PITA, github is very often a PITA, even for folks that have been using git for a long while. I think I understand what your issue is, but can you tell me if you are using the Windows github program? Knowing what platform you're using will make the explanation easier.

**Comment by lottaboost** [comment]
No, I'm using ubuntu.

**Comment by lottaboost** [comment]
When I run a "git stash" this is the output 
[stash.txt](https://github.com/xbmc/repo-plugins/files/428141/stash.txt)

**Comment by lottaboost** [comment]
I can't seem to fix this problem...Now I need to remove the last commit "other persons changes"

**Pull request closed without merging**